boofuzz/primitives/group.py

```diff
@@ -24,8 +24,6 @@ def __init__(self, name, values, default_value=None):
 
         assert len(self.values) > 0, "You can't have an empty value list for your group!"
 
-        if not default_value:
-            default_value = self.values[0]
         self._value = self._original_value = default_value
```

Comment:
@starek4 Thanks. It was supposed to only filter out the `None` value of default_value, not the `''`.

Instead of just removing it, maybe filter out the `None` value in this way.

    if default_value is None:
        default_value = self.values[0]